Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -2933,28 +3068,40 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "121":
-    id: "121"
-    taskid: 6c0ebe20-21a9-4e61-87e5-d4bdc630a92d
-    type: title
+  "132":
+    id: "132"
+    taskid: 0287ece2-86f1-4834-825e-0950ba0f7c42
+    type: regular
     task:
-      id: 6c0ebe20-21a9-4e61-87e5-d4bdc630a92d
+      id: 0287ece2-86f1-4834-825e-0950ba0f7c42
       version: -1
-      name: Analysis Stage
-      type: title
+      name: Set playbook stage grid field (remediation)
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of command:
```

Comment:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Instead of a value, you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of the command:
```